notebooks/utils/cohort_finder.py

```
@@ -0,0 +1,63 @@
+"""Utilities for analyzing patient cohorts in EHR data."""
+from typing import List, Dict, Set, Tuple
```

Ruff

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)